### ISSUE TITLE
Virtualbox is not added to PATH correctly

### ISSUE THREAD

**Issue by Tset-Noitamotua** [ferventcoder/chocolatey-packages#161]
VirtualBox is installed to `C:\Program Files\Oracle\VirtualBox`.

But `choco install virtualbox` adds `C:\PROGRA~2\Oracle\VirtualBox` to PATH.

This requires to manually edit the PATH.

**Comment by ferventcoder** [comment]
Line 13, sure appears it would put it there appropriately. But it doesn't  for some folks - https://chocolatey.org/packages/virtualbox

Let's call it a bug and work on fixing it.

BTW, what version of Chocolatey are you running? This could be something that is already resolved in a newer version of Choco.

**Comment by Tset-Noitamotua** [comment]
My Chocolatey Version is ...
```
λ choco
Chocolatey v0.10.0
```

What about line 9? 

```
if ($is64bit) {$programFiles = ${env:ProgramFiles(x86)}}
``` 

I'm not a powershell expert but doesn't it set `$programFiles` to the 32 bit path (`C:\Program Files (x86)` or short `C:\Progra~2\`) when `$is64bit` is true?

This is exactly what happens in my case. I am on 64bit thus `$programFiles` is set to `C:\Progra~2\` which I then see on PATH. 

PowerShell
------------
```powershell
PS H:\> $is64bit = (Get-WmiObject Win32_Processor).AddressWidth -eq 64
PS H:\> $is64bit
True
PS H:\> $programFiles = $env:programfiles
PS H:\> $programFiles
C:\Program Files
PS H:\> if ($is64bit) {$programFiles = ${env:ProgramFiles(x86)}}
PS H:\> $programFiles
C:\Program Files (x86)
PS H:\>
```

**Comment by Tset-Noitamotua** [comment]
@ferventcoder  plz check my PR

**Comment by ferventcoder** [comment]
Closed through #167
